README.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: HomePage
 # Comment or remove the alert below when it's not needed anymore
-alert: "openHAB 2.5 RC1 Released!"
-alert_link: https://community.openhab.org/t/openhab-milestone-builds/50359/540
-alert_label: Read More
+alert: "openHAB 2.5 Released!"
+# alert_link: https://community.openhab.org/t/openhab-2-5-has-been-released/87716
```

Comment:
why is this in comments now?

Reply:
The link should go to https://www.openhab.org/blog/2019-12-14-openhab-2-5-release.html

Reply:
If the link and label are not explicitely set, it will link to the blog page with a "Read the blog" label:

      <router-link v-else :to="$page.frontmatter.alert_link || '/blog/'" class="alert-button slide">{{$page.frontmatter.alert_label || 'Read the Blog'}} ➜</router-link>